akita/__tests__/entity-store.spec.ts

```diff
@@ -56,6 +56,16 @@ describe('EntitiesStore', () => {
       store.add(new Todo({ id: 1 }));
       expect(store.entities[1]).toBeDefined();
     });
+   
+    it('should prepend with uid', () => {
+      store.add(new Todo({ id: 1 }));
+      store.add(new Todo({ id: 2 }));
+      store.add(new Todo({ id: 3 }));
+      store.add(new Todo({ id: 0 }),{prepend:true});
+      console.log('store.entities',store.entities);
+      
+      expect(store.entities[0]).toBeDefined();
```

Comment:
check that the order is correct

Reply:
You'll find that it's not working as expected. check out this [line](https://github.com/datorama/akita/blob/2c4708d69f0180599174d6064880735642cd1ee3/akita/src/internal/crud.ts#L70).

Reply:
I realize that, if i'm using `QueryConfig` with different order. If I `unshift()` to the list. it will not follow the queryConfig. that can make unpredictable problem.
example that:
```
idKey: 'id'
QueryConfig (orderBy): 'price'
getAll()  :  `[ { id:1, price: 2 }, { id: 2, price: 4 }, { id:3, price: 6 } ]` 
```
if I unshift it with ` { id: 4, price: 1 }`
it will become:
```
getAll() :  `[ { id: 4, price: 1 }, { id:1, price: 2 }, { id: 2, price: 4 }, { id:3, price: 6 } ]` 
```
this will be not correct. So that function it not really useful and could cause the trouble